scripts/sct_dmri_transpose_bvecs.py

```diff
@@ -24,66 +24,71 @@
 
 from __future__ import absolute_import, division
 
+import os
 import sys
-from msct_parser import Parser
+import argparse
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+
 from sct_utils import extract_fname, printv
 import sct_utils as sct
 
 
 # PARSER
 # ==========================================================================================
 def get_parser():
-    # parser initialisation
-    parser = Parser(__file__)
-
-    # # initialize parameters
-    # param = Param()
-    # param_default = Param()
-
     # Initialize the parser
-    parser = Parser(__file__)
-    parser.usage.set_description('Transpose bvecs file (if necessary) to get nx3 structure.')
-    parser.add_option(name='-bvec',
-                      type_value='file',
-                      description='Input bvecs file.',
-                      mandatory=True,
-                      example='bvecs.txt')
-    parser.add_option(name='-i',
-                      type_value='file',
-                      description='Input bvecs file.',
-                      mandatory=False,
-                      example='bvecs.txt',
-                      deprecated_by='-bvec')
```

Comment:
A couple of parameters had two names (e.g. `-i`/`-bvec` here). Argparse doesn't have a deprecation mechanism like this. Do we want to still try to allow usage of the old parameter names? Or, is now a good time to make the switch?

Reply:
definitely a good time to make the switch!